### DISCUSSION TITLE
Noob question on how to sync with gitlab : stucked at the step .env file.

### DISCUSSION THREAD

**Discussion by MacDamien** [200ok-ch/organice#981] [Q&A]
Org-mode seems to be a bit more interesting with the possibility to read and write .org files on the go. Organice is marvelous ! But I’m stopped. When I attempt to connect it says :
Failed to access GitLab project - is the URL correct?

**Comment by MacDamien** [comment]
…

**Comment by munen** [comment]
Hi @MacDamien 

Firstly, thank you for the kind words :pray: 

If you're using the hosted version https://organice.200ok.ch/, then it should work out of the box. If you want to host organice yourself, then here's the documentation on how to set up Gitlab as a synchronization back-end: https://organice.200ok.ch/documentation.html#org253e6d4

Good luck and have fun!
